README.md

```diff
@@ -17,82 +17,100 @@ This version of the library supports the following functionality:
+1. The library handles this request and redirects the user to the Amazon login page, or to the Amazon device's settings page if the user is already logged in.
+1. The user sets up the device in the Amazon UI.
+1. The Amazon LWA redirects the user back to the agent's URL and provides an authorization code.
+1. The agent receives this code and uses it to acquire the required security tokens (Refresh Token and Access Token).
 
-More about authentication [here](https://developer.amazon.com/docs/dash/lwa-web-api.html) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html).
+You can read more about authentication [here](https://developer.amazon.com/docs/dash/lwa-web-api.html) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html).
 
-**Note**: After each restart of the agent the `Refresh Token` should be passed in to the library. So if you don't want to go through the authentication steps again, you may save the token in the agent's persistent storage and set it with the [setRefreshToken()](#setrefreshtokenrefreshtoken) method after each restart of the agent. See [the provided example](#example-store-and-load-refresh-token).
+**Note** The Refresh Token must be passed in to the library every time the agent restarta. If you don't want to go through the above authentication steps again, save the Token in the agent's persistent storage and set it with [*setRefreshToken()*](#setrefreshtokenrefreshtoken) after each agent restart. Please see [the provided example](#examples) for more details.
 
 ### Test Orders ###
 
-For testing purposes, Amazon DRS allows making [test orders](https://developer.amazon.com/docs/dash/test-device-purchases.html). Test orders are those that made by a DRS device authenticated as a test device. So it is determined at the step of authentication whether the device is for testing or not.
-Due to this the [login()](#loginrocky-devicemodel-deviceserial-onauthenticated-route-testdevice) method has a parameter *testDevice*. But if you set a `Refresh Token` manually with the [setRefreshToken()](#setrefreshtokenrefreshtoken) method, only you know whether this token was obtained for testing or not and such *testDevice* parameter is not required here.
+For testing purposes, Amazon DRS allows you to submit [test orders](https://developer.amazon.com/docs/dash/test-device-purchases.html). Test orders are those made by a DRS device authenticated as a test device.
 
-Only test orders can be canceled with the [cancelTestOrder()](#canceltestorderslotid-oncanceled) method.
+As such, [*login()*](#loginrocky-devicemodel-deviceserial-onauthenticated-route-testdevice) has a parameter, *testDevice*, which takes a test device ID. However, if you set a Refresh Token manually with [*setRefreshToken()*](#setrefreshtokenrefreshtoken), only you know whether this token was obtained for testing or not and so *testDevice* is not required in this case.
```

Comment:
*testDevice* parameter takes a boolean value, not ID

Reply:
Fixed